pkg/controller/iam/partialpolicy/iampartialpolicy_controller.go

```diff
@@ -151,6 +151,9 @@ func (r *ReconcileIAMPartialPolicy) Reconcile(ctx context.Context, request recon
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+	if err := r.handleDefaultStateIntoSpecValue(policy); err != nil {
+		return reconcile.Result{}, fmt.Errorf("error handling default values for IAM parital policy '%v': %v", k8s.GetNamespacedName(policy), err)
```

Comment:
Nit: typo in parital.

Nit: In general, we should use %w with fmt.Errorf.  (But not in klog.Warningf, which causes me great frustration!)

Reply:
Done.